File: frontend/nfl-rushing/src/app/domain/rushing-statistic.ts
export class RushingStatistic {
  player = '';
  team = '';
  position = '';
  rushingAttempts = 0;
  rushingAttemptsPerGameAverage = 0;
  rushingFirstDownPercentage = 0;
  rushingFirstDowns = 0;
  rushingFortyYardsEach = 0;
  rushingFumbles = 0;
  rushingTwentyYardsEach = 0;
  rushingYardsAveragePerAttempt = 0;
  totalRushingTouchdowns = 0;
  longestRush = '';
  totalRushingYards = 0;
  yardsPerGame = 0;

  constructor(data: Partial<RushingStatistic> = {}) {
    Object.assign(this, data);
  }
}

File: backend/domain/rushing_statistic.go
package domain

type RushingStatistic struct {
	Player                        string  `json:"Player"`
	Team                          string  `json:"Team"`
	Position                      string  `json:"Pos"`
	RushingAttempts               int     `json:"Att"`
	RushingAttemptsPerGameAverage int     `json:"Att/G"`
	TotalRushingYards             int     `json:"Yds"`
	RushingYarsAveragePerAttempt  float64 `json:"Avg"`
	YardsPerGame                  int     `json:"Yds/G"`
	TotalRushingTouchdowns        int     `json:"TD"`
	LongestRush                   string  `json:"Lng"`
	RushingFirstDowns             int     `json:"1st"`
	RushingFirstDownPercentage    int     `json:"1st%"`
	RushingTwentyYardsEach        int     `json:"20+"`
	RushingFortyYardsEach         int     `json:"40+"`
	RushingFumbles                int     `json:"FUM"`
}
